src/ray/core_worker/test/task_event_buffer_test.cc

```diff
@@ -249,6 +285,68 @@ TEST_F(TaskEventBufferTest, TestFlushEvents) {
   ASSERT_EQ(task_event_buffer_->GetNumTaskEventsStored(), 0);
 }
 
+TEST_F(TaskEventTestWriteExport, TestWriteTaskExportEvents) {
+  // Test writing task events event_EXPORT_TASK_123.log as part of the export API.
```

Comment:
can you add additional tests for

1. batch size being respected
2. if # of events exceed task_events_max_num_export_status_events_buffer_on_worker, it is dropped
?

Reply:
sounds good, added these test cases